app/src/main/java/org/cru/godtools/content/ToolHeader.kt

```diff
@@ -0,0 +1,12 @@
+package org.cru.godtools.content
+
+import androidx.annotation.StringRes
+import org.cru.godtools.R
+
+enum class ToolHeader(
+    @StringRes val descriptionText: Int,
+    @StringRes val openText: Int,
+    @StringRes val dismissText: Int
```

Comment:
I don't want to follow the same pattern as the empty list UI follows. I would have fully self-contained layouts for each potential banner we display.

This enum should then just reference which Banner layout to inflate